### PR TITLE
[FE-1370] Added the Re-verify Domain button to domain details page(Sending, Bounce and Tracking)

### PR DESCRIPTION
**Pull request by shrirupa**
FE-1370 - Added the Re-verify Domain button to domain details page(Sending, Bounce and Tracking)

### What Changed

- Added the Re-Verify Domain Buttons to 
    - Verified Sending / Bounce Domain Page eg - http://localhost:3100/domains/details/sending-bounce/staging-test.spappteam.com
    - Verified Tracking Domain Pages eg - http://localhost:3100/domains/details/tracking/newbob.spappteam.com

### How To Test

- Navigate to any sending/bounce/tracking domain detail pages
- there should be Re-Verify Domain button that should trigger the api to verify these domains
- Should throw appropriate alerts on success/failure

### To Do

- [ ] Address feedback

### CODE REVIEW

File: src/pages/domains/components/TrackingDnsSection.js [hunk 1]
```diff
@@ -71,6 +71,10 @@ export default function TrackingDnsSection({ domain, isSectionVisible, title })
                   </Text>
                   <TranslatableText>record for this domain at your DNS provider.</TranslatableText>
                 </p>
+                <Panel.Action onClick={onSubmit}>
```
Comment (nstanard) [inline src/pages/domains/components/TrackingDnsSection.js:74]: Did we need to add any alerting for if this submit/action call throws/errors?

Comment (shrirupa) [reply src/pages/domains/components/TrackingDnsSection.js:74]: https://github.com/SparkPost/2web2ui/pull/2067/files#diff-40c21c11266b4489c315f67f97e22ffe83d77010b508e68dc3ccec67a55d9b72R80-R83 - added it in the action.

Comment (shrirupa) [reply src/pages/domains/components/TrackingDnsSection.js:74]: I think error alert is currently being handled by Global Alert, do you think we should handle it differently ?

Comment (shrirupa) [reply src/pages/domains/components/TrackingDnsSection.js:74]: I think error alert is currently being handled by Global Alert, do you think we should handle it differently ?

Comment (nstanard) [reply src/pages/domains/components/TrackingDnsSection.js:74]: Nope, just didn't see the handling in the change set and wanted to make sure we didn't need to do anything. Should be pretty easy to test the error cases as we move this too.

**Review by nstanard — APPROVED** [review]
lgtm!

**Pull request merged**